tests/fixtures/src/TokenModel.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * @author Chris Chabilall <chris.c@vanillaforums.com>
+ * @copyright 2009-2018 Vanilla Forums Inc.
+ * @license http://www.opensource.org/licenses/gpl-2.0.php GNU GPL v2
+ */
+
+namespace VanillaTests\Fixtures;
+
+
+use Vanilla\TokenSigningTrait;
+
+/**
+ * Class TokenModel
+ *
+ */
+class TokenModel {
+
+    use TokenSigningTrait;
+    public $tokenIdentifier;
+    public $secret;
+
+    public function __construct($secret = '') {
+        $this->secret = 'sss';
```

Comment:
Same as in the other classes that implement this trait: don't directly access it. Use the setter.